lib/components/inputs/Select/SelectOption/selectOption.scss

```
@@ -22,6 +22,30 @@
   .option-wrapper {
     display: flex;
     align-items: center;
+    width: 100%;
+    min-width: 0;
+
+    .icon-wrapper {
+      flex-shrink: 0;
+      margin-right: 8px;
+    }
+
+    .label-wrapper {
+      min-width: 0;
+      flex: 1;
+
+      .main-label {
```

Since styles are the same, we can list classes together:
```suggestion
      .main-label,
      .sub-label
       {
       ...
       }
```